packages/components/src/SubHeaderBar/TitleSubHeader/TitleSubHeader.module.scss

```diff
@@ -55,6 +55,11 @@ $header-title-max-width: 90rem;
 				line-height: unset;
 				margin: 0;
 			}
+
+			input {
+				width: 30rem;
```

Comment:
Is this fixed value is something that has been reviewed with some UX ? (because i don't understand why 30rem instead of, for example, 20rem or 40rem)

Reply:
this is the value used with the old component, this was reviewed with ux at the time, if i'm not wrong

Reply:
Could we go for `$coral-sizing-maximal` and use the DS tokens, the difference would be to go from 30rem to 32